_proofs/input/button.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Button
+concept: Input
+order: 4
+complexity: 1
+tags: [input, button, mouse]
+---
+For a simple button use a {% api flixel.ui.FlxButton %}. You can call a function when clicked by the mouse.
+
+```haxe
+// create a button that calls onButtonClick when its clicked
+button = new FlxButton(120, 110, "Click me", onButtonClicked);
```

Comment:
Not sure if I should comment it even more to explain the 120, 100 and "Click me". Dunno how basic it should be.

Reply:
Seems fine to me. I wonder if we should use `screenCenter()` instead of manual positioning though (I think that's what most other examples do).

Reply:
If it's being used in other places I'll use it then. Should I update that in the demo too?

Reply:
Yeah